staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -178,6 +178,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		endpointsLister: s.endpointsLister,
 	}
 	s.GenericAPIServer.HandlerContainer.Handle("/apis", apisHandler)
+	s.GenericAPIServer.HandlerContainer.Handle("/apis/", apisHandler)
```

Comment:
Thanks for your review.
I test this pr with both legal and illegal URLs with my patch.
I can't find any difference except when  accessing "/apis/". I thinks this is because the subpath has been registered in other places.
Without this pr when accessing "/apis" we use this handler:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis.go#L77
but when accessing "/apis/"  we use this handler:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/discovery.go#L35
And these tow handlers return different values in my cluster.